esp/services/WsDeploy/WsDeployService.cpp

```diff
@@ -5338,7 +5338,10 @@ bool CWsDeployFileInfo::handleRows(IEspContext &context, IEspHandleRowsRequest &
       }
 
       if (pComponent)
+      {
+        xpath.append("[1]");
```

Comment:
This is to handle a delete on a row where the row is ambiguous.  In case the user erroneously enters the same key-value more than once, this will allow them to delete the duplicate row.